--- Witness/CheckWitness1.cs.opcodes	2018-03-07 08:57:19.901589010 -0300
+++ Witness/CheckWitness1.py.opcodes	2018-03-07 08:58:23.245755882 -0300
@@ -1,39 +1,35 @@
-51	PUSH1	# The number 1 is pushed onto the stack.
+58	PUSH8	# The number 8 is pushed onto the stack.
 c5	NEWARRAY	# 
 6b	TOALTSTACK	# Puts the input onto the top of the alt stack. Removes it from the main stack.
-61	NOP	# Does nothing.
-21	PUSHBYTES33	 # 031a6c6fbbdf02ca351745fa86b9ba5a9452d785ac4f7fc2b7548ca2a46c4fcf4a "lo»ßÊ5Eú¹ºZR×¬OÂ·T¢¤lOÏJ"  # 0x01-0x4B The next opcode bytes is data to be pushed onto the stack
-61	NOP	# Does nothing.
-68	SYSCALL	# Neo.Runtime.CheckWitness
-6c	FROMALTSTACK	# Puts the input onto the top of the main stack. Removes it from the alt stack.
-76	DUP	# Duplicates the top stack item.
-6b	TOALTSTACK	# Puts the input onto the top of the alt stack. Removes it from the main stack.
+42	PUSHBYTES66	 # 303331613663366662626466303263613335313734356661383662396261356139343532643738356163346637666332623735343863613261343663346663663461 "031a6c6fbbdf02ca351745fa86b9ba5a9452d785ac4f7fc2b7548ca2a46c4fcf4a"  # 0x01-0x4B The next opcode bytes is data to be pushed onto the stack
+6a	DUPFROMALTSTACK	# 
 00	PUSH0	#An empty array of bytes is pushed onto the stack
 52	PUSH2	# The number 2 is pushed onto the stack.
 7a	ROLL	# The item n back in the stack is moved to the top.
 c4	SETITEM	# 
-6c	FROMALTSTACK	# Puts the input onto the top of the main stack. Removes it from the alt stack.
-76	DUP	# Duplicates the top stack item.
-6b	TOALTSTACK	# Puts the input onto the top of the alt stack. Removes it from the main stack.
+6a	DUPFROMALTSTACK	# 
 00	PUSH0	#An empty array of bytes is pushed onto the stack
 c3	PICKITEM	# 
-64	JMPIFNOT	# 32
-00	PUSH0	#An empty array of bytes is pushed onto the stack
+68	SYSCALL	# Neo.Runtime.CheckWitness
+61	NOP	# Does nothing.
+6a	DUPFROMALTSTACK	# 
 51	PUSH1	# The number 1 is pushed onto the stack.
-c5	NEWARRAY	# 
-76	DUP	# Duplicates the top stack item.
+52	PUSH2	# The number 2 is pushed onto the stack.
+7a	ROLL	# The item n back in the stack is moved to the top.
+c4	SETITEM	# 
+6a	DUPFROMALTSTACK	# 
+51	PUSH1	# The number 1 is pushed onto the stack.
+c3	PICKITEM	# 
+64	JMPIFNOT	# 28
 00	PUSH0	#An empty array of bytes is pushed onto the stack
 0f	PUSHBYTES15	 # 4f574e45522069732063616c6c6572 "OWNER is caller"  # 0x01-0x4B The next opcode bytes is data to be pushed onto the stack
-c4	SETITEM	# 
-61	NOP	# Does nothing.
-68	SYSCALL	# Neo.Runtime.Notify
+68	SYSCALL	# Neo.Runtime.Log
 51	PUSH1	# The number 1 is pushed onto the stack.
-61	NOP	# Does nothing.
 6c	FROMALTSTACK	# Puts the input onto the top of the main stack. Removes it from the alt stack.
 75	DROP	# Removes the top stack item.
 66	RET	# 
-00	PUSH0	#An empty array of bytes is pushed onto the stack
 61	NOP	# Does nothing.
+00	PUSH0	#An empty array of bytes is pushed onto the stack
 6c	FROMALTSTACK	# Puts the input onto the top of the main stack. Removes it from the alt stack.
 75	DROP	# Removes the top stack item.
 66	RET	# 
